03-account-kit-manage-resources/index.ts

```diff
@@ -0,0 +1,37 @@
+// SDK exposes all you need
+import { Chains } from '@[SOMETHING]kit'
+import { Asset, RamBytes, Account, SigningRequest } from '@[SOMETHING]kit/account'
+
+async function main() {
+    // Loads account
+    const account: Account = await Account.load("teamgreymass", Chains.EOS)
+
+    // Buy ram
+    const buyRamEsr: SigningRequest = account.actions.buyRam(1000)
```

Comment:
I don't think any of these are going to return ESRs, or are they? I kinda figured they'd return actions and/or transactions 🤔 

Let's discuss.

Reply:
I like the idea of them returning ESRs since it allows us to decouple the concerns of the Account and Session kits a bit.

Reply:
That's a lot of encoding/decoding though, when we could probably just use `Transaction` objects to achieve the same sort of thing.